.gitlab-ci.yml

```diff
@@ -2,11 +2,28 @@
 include:
   - 'https://git.scc.kit.edu/m-team/ci-voodoo/raw/master/ci-include/generic-ci.yml'
   - 'https://git.scc.kit.edu/m-team/ci-voodoo/raw/master/ci-include/pipeline-jobs.yml'
+  - 'https://git.scc.kit.edu/m-team/ci-voodoo/raw/master/ci-include/pipeline-jobs-publish-to-repo.yml'
 
 variables:
   STAGING_BRANCH_NAME: 'staging'
   DOCKER_IMAGE_NAMESPACE: 'marcvs/build'
   DOCKER_IMAGE_NAME: 'oidc-agent'
+  GIT_STRATEGY: clone
+  GIT_DEPTH: 0
+  #TARGET_REPO: 'devel'
+  # The following varialbes can be overwritten only in specific targets
+  # See generic-ci.yml -> .repo-definition for the full list
+  # REPO_BASE => deprecated
+  # REPO_USER
+  # REPO_HOST
+  # PUBLISH_BUILD_RESULTS_REPO: 'devel'
+  # PUBLISH_TESTED_RESULTS_REPO: 'prerel'
+  # PUBLISH_MASTER_RESULTS_REPO: 'for-prod'
+  # PREREL_BRANCH_NAME: 'prerel'
+  # See generic-ci.yml -> .build-definition for the full list
+  # DOCKER_IMAGE_VERSION
+  # DOCKER_IMAGE_VERSION_WINDOWS
+  PREREL_BRANCH_NAME: 'ci/fix-new-ci'
```

Comment:
I guess this should be changed to `prerel` or commented out, depending on how things work

Reply:
Done